.github/workflows/autobuild.yml

```diff
@@ -408,23 +408,24 @@ jobs:
                                     needs.create_release.outputs.publish_to_release == 'true' &&
                                     steps.build.outputs.macos_notarize == 'true'
         id:                         notarize-macOS-app
-        uses:                       lando/notarize-action@4f5869b09386e8336802159031e4189e0919ae20
-        with:
-          product-path:             deploy/${{ steps.get-artifacts.outputs.artifact_1 }}
-          primary-bundle-id:        io.jamulus.Jamulus
```

Comment:
Where has the replacement for the bundle ID moved to in the refactor?

Reply:
It's no longer needed. We have it in Jamulus.pro now